libs/ui-lib/lib/common/components/ui/OpenShiftVersionDropdown.tsx

```diff
@@ -47,15 +58,19 @@ export const OpenShiftVersionDropdown = ({
   const isDisabled = versions.length === 0;
 
   const { defaultLabel, defaultValue } = React.useMemo(() => {
```

Comment:
With these changes, `defaultLabel / defaulValue` are not default values anymore, as they also change with the selection.

This can be problematic as it's mixing with `current`. I think they should still be separate things.